platform/ios/jazzy.yml

```diff
@@ -23,53 +26,61 @@ custom_categories:
       - MGLMapViewDelegate
       - MGLStyle
       - MGLUserTrackingMode
-  - name: Annotations
+  - name: User Interaction
     children:
-      - MGLAnnotation
-      - MGLAnnotationImage
-      - MGLAnnotationView
       - MGLCalloutView
       - MGLCalloutViewDelegate
+  - name: Protocols and Abstract Classes
+    children:
+      - MGLAnnotation
```

Comment:
This protocol is unlike the others in this category in that the developer can actually make any class they want conform to it, and we’ll treat that class just like an MGLPointAnnotation. I think it should go in the “Annotations” category alongside MGLAnnotationImage and MGLAnnotationView.